### ISSUE TITLE
Some icons appears in duplicate with Firefox 55

### ISSUE THREAD

**Issue by baydam** [horst3180/arc-firefox-theme#131]

**Comment by ghost** [comment]
Second this. Note the duplicated Pocket icon, the duplicated forward backward buttons and lack of refresh button.

![screenshot from 2017-08-10 10-22-23](https://user-images.githubusercontent.com/28416528/29180716-dd67e270-7db5-11e7-9eec-7ad1a802a9d0.png)

**Comment by Narrat** [comment]
And in the menu the duplicated Sync-Panel

**Comment by ghost** [comment]
The sync duplication is bug #123, and may be unrelated. I noticed that sync was not functioning properly (with duplicated icon in sync panel) on 54 before I updated to 55, and the other weirdness occurred after upgrade to 55.

**Comment by Narrat** [comment]
Hm, okay. I just noticed it with FF55, but possible I overlooked it. Since I only looked for quirks after noticing issues Forward/Backward and reload.
Thanks for the pointer

**Comment by Siphonay** [comment]
I face those issues too, seems that the theme engine have changed a little with Firefox 55, meaning Arc requires an update. I've changed to the Compact Dark theme until then

**Comment by ArchangeGabriel** [comment]
This is partly https://github.com/horst3180/arc-firefox-theme/issues/121 and in the end mostly https://github.com/horst3180/arc-firefox-theme/issues/120. Please read the thread if you want more details, but else:
**TL;DR:** Firefox themes are dead.

**Comment by sirfz** [comment]
There was an issue with duplicate info in Firefox 54 (#124) but there hasn't been a FF 54 arc-theme release since 53 so it seems development hasn't been active since then.

**Comment by ArchangeGabriel** [comment]
@sirfz Sorry if I wasn’t clear: it’s normal that there hasn’t been any active development, because they can’t be any, Firefox **REMOVED** support for theming.

**Comment by ArchangeGabriel** [comment]
(To be more precise, they’re removing parts by parts)

**Comment by sirfz** [comment]
@ArchangeGabriel from what I understand, that applies to the coming Firefox 57 release which features the new GUI (Photon UI) but not the current release. Also, the desktop Arc-theme hasn't been active either horst3180/arc-theme#810

**Comment by ArchangeGabriel** [comment]
@sirfz No, they’re landing Photon UI parts by parts starting from Firefox 54. In 57 what happens is that you can’t even try to load another theme. Currently you still can, but they might be broken depending on which elements they are theming.

**Comment by jgillich** [comment]
FWIW they blogged about this in February: https://blog.mozilla.org/addons/2017/02/24/improving-themes-in-firefox/
